app/Resources/translations/messages.nl.yml

```diff
@@ -0,0 +1,55 @@
+service_provider: Service provider
+
+page.title: GSSP SDK
+
+page.header.stepup-gateway-gssp: Wiki
+page.header.stepup-gssp-bundle: GSSP-bundel
+
+page.navigation.route.homepage: Home
+page.navigation.route.app_identity_registration: Registratie
+page.navigation.route.app_identity_authentication: Authenticatie
+page.navigation.route.gssp_saml_sso: SSO
+page.navigation.route.gssp_saml_sso_return: SSO return
+page.navigation.route.gssp_saml_metadata: Metadata
+
+page.footer.terms: Algemene voorwaarden
+
+page.index.title: Welkom bij Generieke SAML Stepup Provider SDK
+page.index.body.intro: |
+    Je applicatie is klaar voorgebruik. U kunt dit voorbeeld gebruiken om uw verificatie te implementeren.
+    SDK is geconfigureerd met demo-configuratie en kan registratie en authenticatie bieden voor de demo SP.
+
+page.index.body.sdk_discription: |
+    De AppBundle bevat de voorbeeldregistratie en verificatie-endpoints. Dit is het hoofddoel voor uw toepassing. Zie DefaultController::registrationAction and DefaultController::authenticateAction.
+    Dit is de plaats om de logica van je IdP te implementeren. Lees voor meer informatie de README.md van dit project.
+
+page.index.body.registration.title: Registratie flow
+page.index.body.registration.step1.go_to_sp: 1) Ga naar
+page.index.body.registration.step2.select_idp: 2) Selecteer '%idp-entity-id%' in de lijst 'Identity Provider'
+page.index.body.registration.step3.click_login: 3) Klik op de knop 'Aanmelden'
+page.index.body.registration.step4.click_register: 4) Klik op de knop 'Gebruiker registreren'
+
+page.index.body.authentication.title: Verificatie flow
+page.index.body.authentication.step1.go_to_sp: 1) Ga naar
+page.index.body.authentication.step2.select_idp: 2) Selecteer '%idp-entity-id%' in de lijst 'Identity Provider'
+page.index.body.authentication.step3.fill_subject: 3) Vul de identity NameId in het veld 'subject' in
+page.index.body.authentication.step4.click_login: 4) Klik op de knop 'Aanmelden'
+page.index.body.authentication.step5.click_authenticate: 5) Klik op de knop 'Gebruiker verifiëren'
+
+page.route.description.app_identity_registration: |
+    Deze url is in het menu gekoppeld voor demo en documentatie doeleinden.
+    Het applicatiepad dat de registratie afhandelt, dit is het startpunt om de aangepaste registratielogica te implementeren.
+page.route.description.app_identity_authentication: |
+    Deze url is in het menu gekoppeld voor demo en documentatie doeleinden.
+    Het applicatiepad dat de authenticatie afhandelt, dit is het toegangspunt om de aangepaste authenticatielogica te implementeren.
+page.route.description.gssp_saml_sso: |
+    Deze url is in het menu gekoppeld voor demo en documentatie doeleinden.
+    Het wordt geleverd door de GSSP-bundel die de aanvraag Authnequest saml van de serviceprovider afhandelt.
```

Comment:
- ~~Authnequest~~ AuthnRequest of AuthNRequest of vertalen naar authenticatie request?
- saml met hoofdletters schrijven?